src/host/ut_host/ScreenBufferTests.cpp

```diff
@@ -1293,7 +1293,7 @@ void ScreenBufferTests::VtResizeDECCOLM()
     newViewWidth = si.GetViewport().Width();
 
     VERIFY_IS_FALSE(areMarginsSet());
-    VERIFY_ARE_EQUAL(COORD({ 0, 0 }), getRelativeCursorPosition());
+    VERIFY_ARE_EQUAL(til::point(0, 0), getRelativeCursorPosition());

```

Comment:
The `VERIFY_ARE_EQUAL` macro blows up if you use curly brackets inside it. Not kidding. IIRC the `({ 0, 0 })` doesn't work because the constructor is explicit and doesn't take a single parameter list argument (unlike structs).